transformations/lost_in_translation/transformation.py

```diff
@@ -0,0 +1,110 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import random
+from transformers import pipeline
+
+class LostInTranslation(SentenceOperation):
+  '''
+  Repeated translation of input text
+  '''
+  tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+  languages = ["en"]
+  heavy = True
+
+  def __init__(self, seed=0,
+               langs=['es', 'de', 'zh', 'fr', 'ru'],
+               how='strict',
+               layers=5,
+               max_length=600,
+               max_outputs=1,
+               device=-1,
+               ):
+    '''
+    seed: set to None to disable
+    langs: list of languages, from those supported by OpusMT
+    how:
+      "strict" for static list order,
+      "shuffle" to shuffle said list,
+      "random" to draw (layers) times with replacement
+    device: use CPU (-1) or GPU (>=0)
+    layers: number of encode/decode operations to perform
+    max_length: this is passed to pipeline
+    max_outputs: this is passed to super
+    '''
+    super().__init__(seed,max_outputs)
+    self.langs = langs
+    self.how = how
+    self.device = device
+    self.layers = layers
+    self.max_length = max_length
+
+  def encode_decode(self, text, lang, max_length=600, device=-1):
+    # translate and un-translate
+    # using Helsinki-NLP OpusMT models
+    encode = pipeline(
+        'translation_en_to_{}'.format(lang),
+        model="Helsinki-NLP/opus-mt-en-{}".format(lang),
```

Comment:
Also, you might want to add in the README why you've chosen these models & their papers if available.